<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;배달의민족 주아&quot;;</string>
   </property>
   <property name="text">
    <string>여행지</string>
   </property>
  </widget>
  <widget class="QComboBox" name="location_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;배달의민족 주아&quot;;</string>
   </property>
   <property name="text">
    <string>여행 기간</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;배달의민족 주아&quot;;</string>
   </property>
   <property name="text">
    <string>여행 인원</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;배달의민족 주아&quot;;</string>
   </property>
   <property name="text">
    <string>옵션 추가 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="wishlist_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>610</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Arca Majora 3 Bold&quot;;</string>
   </property>
   <property name="text">
    <string>Add to Wish List</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>성인</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="adult_spinBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>어린이</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="child_spinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbl1_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;배달의민족 주아&quot;;</string>
   </property>
   <property name="text">
    <string>당신의 여행 위시 리스트</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="startDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>5</month>
     <day>25</day>
    </datetime>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="endDate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>5</month>
     <day>2</day>
    </datetime>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>481</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>여행지</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="location_line"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>여행기간</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="startdate_line"/>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>~</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="enddate_line"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>여행 인원</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>25</number>
      </property>
      <item>
       <widget class="QLineEdit" name="adult_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="child_line"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="budget_line"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>추가 옵션</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="carRent_btn">
      <property name="minimumSize">
       <size>
        <width>197</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>렌터카 서비스 포함</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="carRent_line">
      <property name="text">
       <string>미포함</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="hotel_btn">
      <property name="minimumSize">
       <size>
        <width>197</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>호텔 숙박 포함</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>미포함</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="tour_btn">
      <property name="minimumSize">
       <size>
        <width>197</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>가이드 투어 포함</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string>미포함</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
